### PR TITLE
Remove semantic-ui (pt 2)

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: client/src/components/common/common.js [hunk 1]
```diff
@@ -3,45 +3,43 @@ import {
...
 import Icon from './icon/icon';
+import IconButton from './icon-button/icon-button';
+import Input from './input/input';
+import Message from './message/message';
 import Notifications from './notifications/notifications';
 import Post from './post/post';
 import PrivateRoute from './private-route/private-route';
 import PublicRoute from './public-route/public-route';
 import Spinner from './spinner/spinner';
+import ToggleCheckbox from './toggle-checkbox/toggle-checkbox';
```
Comment (what1s1ove) [inline client/src/components/common/common.js:22]:
```suggestion
import Checkbox from './checkbox/checkbox';
```
will be enough

File: client/src/components/common/icon-button/icon-button.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+import styles from './styles.module.scss';
+
+const IconButton = ({
+  icon,
+  label,
+  onClick
+}) => (
+  <button className={styles.iconButton} type="button" onClick={onClick}>
+    {icon}
```
Comment (what1s1ove) [inline client/src/components/common/icon-button/icon-button.jsx:11]:
```suggestion
    <Icon name={iconName} />
```

File: client/src/components/common/input/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  border: 1px solid rgba(34,36,38,.15);
+  border-radius: 0.25rem;
+  box-shadow: none;
+  font-weight: 400;
+  outline: none;
+  padding: 0.6rem 1rem;
+
+  &.withIcon {
+    /*
+    * "!important" used to rewrite semantic ui form passed style
```
Comment (what1s1ove) [inline client/src/components/common/input/styles.module.scss:30]: do it

File: client/src/components/common/notifications/common.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const NotificationMessage = {
+  LIKED_POST: 'Your post was liked!'
+};
+
+const SocketEvent = {
```
Comment (what1s1ove) [inline client/src/components/common/notifications/common.js:5]: hmmm
we have already spilt it up

File: client/src/components/common/toggle-checkbox/toggle-checkbox.jsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+
+import styles from './styles.module.scss';
+
+const ToggleCheckbox = ({ checked, label, onChange }) => (
```
Comment (what1s1ove) [inline client/src/components/common/toggle-checkbox/toggle-checkbox.jsx:5]:
```suggestion
const Checkbox = ({ checked, label, onChange }) => (
```
don't forget about name

File: client/src/components/common/input/input.jsx [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+const Input = forwardRef(({
+  disabled,
+  icon,
+  placeholder,
+  type,
+  value,
+  ...fieldControls
+}, ref) => (
+  <div className={styles.inputContainer}>
+    {icon && <span className={styles.icon}>{icon}</span>}
```
Comment (what1s1ove) [inline client/src/components/common/input/input.jsx:15]:
```suggestion
    {iconName && <span className={styles.icon}><Icon name={iconName} /></span>}
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: client/src/components/common/notifications/common.js (hunk 4 above)
Comment (andanfe) [reply client/src/components/common/notifications/common.js:5]: My bad, yesterday's PR was cherry-picked from this branch's commit, so this file just wasn't deleted :c

File: client/src/components/common/copy-buffer-input/copy-buffer-input.jsx [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+import Icon from '../icon/icon';
+
+import styles from './styles.module.scss';
+
+const CopyBufferInput = forwardRef(({ onCopy, value }, ref) => (
+  <div className={styles.copyContainer}>
+    <input ref={ref} className={styles.copyInput} type="text" value={value} disabled />
+    <button className={styles.copyBtn} type="button" onClick={onCopy}>
+      Copy
+      <span className={styles.icon}><Icon name="copy" /></span>
```
Comment (Liza-Veis) [inline client/src/components/common/copy-buffer-input/copy-buffer-input.jsx:13]:
```suggestion
      <span className={styles.icon}><Icon name={IconName.COPY} /></span>
```

**Review by liza-veis — APPROVED** [review]

File: client/src/components/common/icon-button/icon-button.jsx [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+import styles from './styles.module.scss';
+
+const IconButton = ({
+  iconName,
+  label,
+  onClick
+}) => (
+  <button className={styles.iconButton} type="button" onClick={onClick}>
+    {<Icon name={iconName} />}
```
Comment (what1s1ove) [inline client/src/components/common/icon-button/icon-button.jsx:14]:
```suggestion
    <Icon name={iconName} />
```

File: client/src/components/common/checkbox/checkbox.jsx [hunk 9]
```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+
+import styles from './styles.module.scss';
+
+const Checkbox = ({ checked, label, onChange }) => (
```
Comment (what1s1ove) [inline client/src/components/common/checkbox/checkbox.jsx:5]:
```suggestion
const Checkbox = ({ isChecked, label, onChange }) => (
```

**Review by ghost — APPROVED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**